proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/saml/HubResponseTranslator.java

```diff
@@ -87,6 +92,30 @@ public Response translate(HubResponseContainer hubResponseContainer) {
         );
     }
 
+    private void validateHubResponseContainerAttributes(HubResponseContainer hubResponseContainer) {
+        if ( hubResponseContainer == null ) {
+            throw new HubResponseTranslationException("HubResponseContainer is null.");
+        }
+        if (hubResponseContainer.getVspScenario() != null && hubResponseContainer.getVspScenario().equals(VspScenario.IDENTITY_VERIFIED)) {
```

Comment:
Could we simplify this to:
`VspScenario.IDENTITY_VERIFIED == hubResponseContainer.getVspScenario()`